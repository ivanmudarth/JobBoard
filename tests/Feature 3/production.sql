-- Feature 3

-- User ivanm(id = 3) adds two replys to job posting with id=2
INSERT INTO Reply VALUES (2, 3, 'this is my first reply', '2023-3-5 14:56:59')
INSERT INTO Reply VALUES (2, 3, 'this is my second reply', '2023-3-5 14:58:00')

SELECT * FROM Reply

-- View all replies to job posting with id=2
SELECT user_id, reply_text 
FROM Reply 
WHERE job_posting_id = 2 
ORDER BY date_posted DESC;