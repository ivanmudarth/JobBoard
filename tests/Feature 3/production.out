Execute:
> SELECT * FROM Reply

+ ------------------- + ------------ + --------------- + -------------- +
| job_posting_id      | user_id      | reply_text      | timestamp      |
+ ------------------- + ------------ + --------------- + -------------- +
| 2                   | 3            | this is my first reply | 2023-03-05 14:56:59 |
| 2                   | 3            | this is my second reply | 2023-03-05 14:58:00 |
+ ------------------- + ------------ + --------------- + -------------- +
2 rows


Execute:
> SELECT user_id, reply_text 
FROM Reply 
WHERE job_posting_id = 2 
ORDER BY timestamp DESC

+ ------------ + --------------- +
| user_id      | reply_text      |
+ ------------ + --------------- +
| 1            | this is my second reply |
| 1            | this is my first reply |
+ ------------ + --------------- +
2 rows